### PR TITLE
Fix General installer components

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #68*

*Description of changes:*
- Added TDC to installer package
- Updated Driver as a required component in the installer
- Changed the binary & project name

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -57,7 +57,7 @@ endif()
 
 # Base directories
 set(PROJECT_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/..")
-set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/elasticodbc")
+set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/odfesqlodbc")
 set(ESENLIST_SRC "${CMAKE_CURRENT_SOURCE_DIR}/elasticenlist")
```
Comment (jordanw-bq) [inline src/CMakeLists.txt:61]: Rename this project as well (something like `ODFEENLIST_SRC`)

File: src/odfesqlodbc/es_communication.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 #include "es_communication.h"
 
-// elasticodbc needs to be included before mylog, otherwise mylog will generate
+// odfesqlodbc needs to be included before mylog, otherwise mylog will generate
 // compiler warnings
 // clang-format off
 #include "es_odbc.h"
```
Comment (jordanw-bq) [inline src/odfesqlodbc/es_communication.cpp:22]: Rename header file as well

File: src/CMakeLists.txt [hunk 3]
```diff
@@ -57,7 +57,7 @@ endif()
 
 # Base directories
 set(PROJECT_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/..")
-set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/elasticodbc")
+set(ESODBC_SRC "${CMAKE_CURRENT_SOURCE_DIR}/odfesqlodbc")
```
[the remark this answers is not in the document]
Comment (rupal-bq) [reply src/CMakeLists.txt]: changed

File: src/CMakeLists.txt (hunk 1 above)
Comment (rupal-bq) [reply src/CMakeLists.txt]: Renamed project

**Pull request merged**